code/game/jobs/job/civilian.dm

```diff
@@ -303,8 +303,52 @@
 	shoes = /obj/item/clothing/shoes/brown
 	glasses = /obj/item/clothing/glasses/sunglasses/big
 	l_ear = /obj/item/device/radio/headset/ia
+
 	l_hand =  /obj/item/weapon/storage/briefcase
 
 	implants = list(
 		/obj/item/weapon/implant/loyalty
+	)
+
+/datum/job/comedian
+	title = "Entertainer"
+	flag = CLOWN
+	department = "Civilian"
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the head of personnel"
+	selection_color = "#fe019a"
+	access = list(access_maint_tunnels)
+	minimal_access = list(access_maint_tunnels)
+	alt_titles = list("Clown")
+	alt_outfits = list("Clown" = /datum/outfit/job/clown)
+	title_accesses = list("Clown" = list(access_clown), "Entertainer" = list(access_medical, access_sec_doors, access_research, access_engine))
+	outfit = /datum/outfit/job/entertainer
+
+/datum/outfit/job/entertainer
+	name = "Entertainer"
+	jobtype = /datum/job/comedian
+
+	uniform = /obj/item/clothing/under/lightblue
+	pda = /obj/item/device/pda/mime

```

Comment:
I'm concerned that this is clown powercreep. They need the honk cartridge, not the mime cartridge.